### PR TITLE
Update for the rename doc-builder -> hf-doc-utils

### PR DESCRIPTION
**Pull request by sgugger**
This PR adapts the job to the upcoming change of name of `doc-builder`.

### CODE REVIEW

File: .github/workflows/build_dev_documentation.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
   build_and_package:
     runs-on: ubuntu-latest
     container:
-      image: huggingface/transformers-doc-builder
+      image: huggingface/transformers-hf-doc-utils
```
Comment (mishig25) [inline .github/workflows/build_dev_documentation.yml:14]: was this `image` also renamed?

**Comment by julien-c** [comment]
why utils? it's a builder no?

**Comment by mishig25** [comment]
@julien-c there was a vote 🙂 https://huggingface.slack.com/archives/C021H1P1HKR/p1646405136644739

File: .github/workflows/build_dev_documentation.yml (hunk 1 above)
Comment (sgugger) [reply .github/workflows/build_dev_documentation.yml:14]: No, it's a mistake on my part :-)

**Comment by lhoestq** [comment]
Thanks ! It looks all good to me (provided `hf-doc-utils` is the name we keep in the end). I'm fine with this name, and `hf-doc-builder` is also fine IMHO

**Comment by julien-c** [comment]
ok, this is definitely not a hill I'll die on =) @mishig25 @sgugger

**Pull request closed without merging**